libphal/phal_dump.C

```diff
@@ -186,52 +192,62 @@ struct pdbg_target* preCollection(const uint32_t failingUnit,
 		throw std::runtime_error("libekb initialization failed");
 	}
 
-	// FSI and PIB targets for executing HWP
-	struct pdbg_target *fsi, *pib;
-	char path[16];
-
-	// Find the proc target from the failing unit id
-	struct pdbg_target* proc = getProcFromFailingId(failingUnit);
+	constexpr int ODYSSEY_SBE_DUMP = 0xB;
```

Comment:
keep these consts in top of this file or in an header file for both ODYSSEY_SBE_DUMP and PROC_SBE_DUMP , use if else based on sbeTypeId